pkgs/top-level/all-packages.nix

```diff
@@ -23373,7 +23373,7 @@ with pkgs;
 
   vte = callPackage ../development/libraries/vte {
     # Needs GCC ≥10 but aarch64 defaults to GCC 9.
-    stdenv = clangStdenv;
+    stdenv = if stdenv.isLinux then gcc12Stdenv else stdenv;
```

Comment:
Why not a longer condition, e.g. like the one above?  I mean, if the comment is the reason, why override in the typical x86_64-linux case?

Reply:
Restricted now to aarch64-linux.